airbyte-webapp/src/components/ConnectorBlocks/FormPageContent.tsx

```diff
@@ -1,6 +1,7 @@
 import styled from "styled-components";
 
 const FormPageContent = styled.div<{ big?: boolean }>`
+  width: 80%;
```

Comment:
I tested but could not find the impact of this 80% width setting. What does it fix?

Reply:
It keeps the forms on `Create Source` and `Create Destination` from taking up the full panel width when the docs are open.  To see: go to "Sources", "Add a Source", select a connector, click "Setup Guide".  Now if you comment out this line, you'll see the form takes up the full div (depending on screen size).  Sounds like there are some other things to revisit with this form's styling when the docs are open though as well.